internal/cmd/rc.go

```diff
@@ -388,6 +388,9 @@ func findEnvUp(searchDir string, loadDotenv bool) (path string) {
 }
 
 func findUp(searchDir string, fileNames ...string) (path string) {
+	if searchDir == "" {
+		return ""
```

Comment:
i tested and this actually works fine without the explicit return, but i was worried that it would search parent directories or something like that.